ompi/mpi/java/java/MPI.java

```diff
@@ -204,7 +204,15 @@
 
 	static
 	{
-		System.loadLibrary("mpi_java");
+		try
+                {
+                    System.loadLibrary("mpi_java") ;
+                }
+                catch (UnsatisfiedLinkError e)
+                {
+                    System.err.println("mpi java lib failed to load" + e  + "\n") ;
```

Comment:
Please insert a space of something before the Java exception message.

```suggestion
                    System.err.println("mpi java lib failed to load: " + e  + "\n") ;
```